src/composables/notifications.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  computed, onBeforeUnmount, onMounted, ref,
+} from '@vue/composition-api';
+import {
+  IAccount, IChainNames,
+  INetwork,
+  INotification,
+  INotificationSetting,
+  NotificationStatus,
+} from '../types';
+import { useGetter, useState } from './vuex';
+import {
+  NOTIFICATION_STATUS_CREATED,
+  NOTIFICATION_STATUS_READ,
+  NOTIFICATION_TYPE_WALLET,
+  executeAndSetInterval,
+  watchUntilTruthy, fetchJson, postJson, NOTIFICATION_ENTITY_TYPE_TIP, AGGREGATOR_URL,
+} from '../popup/utils';
+
+const fetchedNotificationsLimit = 20;
+
+const mountedComponentsCounter = ref<number>(0);
+const notificationsSuperhero = ref<INotification[]>([]);
+const notificationsWallet = ref<INotification[]>([]);
+let pollingInterval: NodeJS.Timer | null = null;
```

Comment:
`setTimeout` return "timer" or so called "intervalId". So I suggest to name this variable `pollingTimer` or (my choice) `pollingIntervalId`.